operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -412,6 +412,22 @@ mimir:
   # -- Additional structured values on top of the text based 'mimir.config'. Applied after the text based config is evaluated for templates. Enables adding and modifying YAML elements in the evaulated 'mimir.config'.
   structuredConfig: {}
 
+  # Example
```

Comment:
we're still not generic a readme, but if we put this above the `structuredConfig` item above it will be working with [helmdocs](https://github.com/norwoodj/helm-docs)

Reply:
Tx I'll fix it tomorrow :)

Reply:
Fixed :)